services/ses/pod-ses/src/main.ts

```diff
@@ -22,25 +23,39 @@ import { SES } from './ses'
 import { Endpoint } from './types'
 
 const errorMessages = ['expired', 'Unregistered', 'No such subscription']
-async function sendPushToSubscription (subscription: PushSubscription, data: PushData): Promise<'ok' | 'clear-push'> {
-  try {
-    await webpush.sendNotification(subscription, JSON.stringify(data))
-  } catch (err: any) {
-    if (err instanceof WebPushError) {
-      if (errorMessages.some((p) => JSON.stringify((err as WebPushError).body).includes(p))) {
-        return 'clear-push'
+async function sendPushToSubscription (
+  subscriptions: PushSubscription[],
+  data: PushData
+): Promise<Ref<PushSubscription>[]> {
+  const result: Ref<PushSubscription>[] = []
+  for (const subscription of subscriptions) {
+    try {
+      await webpush.sendNotification(subscription, JSON.stringify(data))
```

Comment:
can we send them in parallel?

Reply:
It is for one user, I think no need, we send in parallel for different users.